site2/docs/reference-configuration.md

```diff
@@ -4,14 +4,6 @@ title: Pulsar configuration
 sidebar_label: "Pulsar configuration"
 ---
 
-You can manage Pulsar configuration by configuration files in the [`conf`](https://github.com/apache/pulsar/tree/master/conf) directory of a Pulsar [installation](getting-started-standalone.md).
+You can manage Pulsar configuration by configuration files in the [`conf`](https://github.com/apache/pulsar/tree/master/conf) directory of a Pulsar [installation](https://pulsar.apache.org/docs/next/getting-started-standalone).
```

Comment:
The reason for this change is the same to https://github.com/apache/pulsar/pull/17197#discussion_r951162040?

I guess no because `reference-configuration.md` is still in the same repo as `getting-started-standalone.md`, `getting-started-standalone/` is still valid, correct?

Reply:
No, link `getting-started-standalone.md` is not valid here because here is another site, not the Docusaurus site. But the "Markdown Link" syntax here is Docusaurus feature.

Reply:
Oh, I see (as below)
<img width="614" alt="image" src="https://user-images.githubusercontent.com/50226895/185897958-57f4cc6e-9f17-4a38-90e2-a8dffd4aadcb.png">

Sorry I got the site mixed up 😝

Reply:
<img width="1727" alt="image" src="https://user-images.githubusercontent.com/18818196/188595386-3cf2cf17-2c9d-41a4-9425-de1feb341bf7.png">

No. They're under the same site. I'll handle this alongside with fixing all back links to `getting-started-standalone.md`